[Node.js] Style guide fixes for TDBX vale rules (#842)
mongodb/docs-node: source/usage-examples/command.txt

   The JavaScript and TypeScript code snippets above are identical. There are no
   TypeScript specific features of the driver relevant to this use case.

> Running the preceding command, you see the following output:

Reviewer: S: I think the copy on all of the usage examples should match:
```suggestion
Running the preceding example results in the following output:
```

Author: Leaving as is to match the other occurences.

> When you run the preceding command, you will see the following output:

Reviewer: ```suggestion
Running the preceding example results in the following output:
```
